src/contributing.md

```diff
@@ -320,7 +320,7 @@ reveals what the problem is:
 
 > Next up we need to ensure that our lock file is updated to use this new
 > version of uuid so our project uses the locally checked out copy instead of
-> one from crates.io. The way [patch] works is that it'll load the dependency
+> one from crates.io. The way `\[patch\]` works is that it'll load the dependency
```

Comment:
You only need one or the other, this will render like this:

`\[patch\]`

Reply:
Hmm... I remember when I tried this locally it broke :/ I'll have to take a look again later...

Reply:
I am now unable to reproduce the original error... looks like it was fixed in the latest version of mdbook?